src/app/tests/suites/certification/Test_TC_AIRQUAL_2_2.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 3.2.1. [TC-AIRQUAL-2.2] Attributes with server as DUT
+
+PICS:
+    - AIRQUAL.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Air Quality"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads from the DUT the AirQuality attribute."
+      PICS: AIRQUAL.S.A0000
+      command: "readAttribute"
+      attribute: "AirQuality"
+      response:
+          saveAs: ValueBeforeChange
+          constraints:
+              type: enum8
+              minValue: 0
+              maxValue: 6
+
+    - label: "Operate on device to change the air quality significantly"
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT && AIRQUAL.M.AirQualityChange
```

Comment:
This and the bits after it should have PICS_SKIP_SAMPLE_APP in the PICS conditions.

Reply:
I think the test will be removed entirely since it tests actual device functionality which should not be required for Matter.